packages/data-explorer/tests/unit/mappers/mapperUtils.spec.ts

```diff
@@ -0,0 +1,103 @@
+import { getCategoricals, generateUri, getFieldOptions } from '@/mappers/mapperUtils'
+import meta from '../mocks/metaDataResponseMock'
+import metatypes from '../mocks/metaDataTypesResponseMock'
+import { MetaDataAttribute } from '@/types/ApiResponse'
+// @ts-ignore
+import api from '@molgenis/molgenis-api-client'
+
+jest.mock('@molgenis/molgenis-api-client', () => ({
+  get: jest.fn().mockResolvedValue({ 'href': '/api/v2/root_gender', 'meta': { 'href': '/api/v2/root_gender', 'hrefCollection': '/api/v2/root_gender', 'name': 'root_gender', 'label': 'Gender', 'description': 'gender is located in the root package because it is not hospital specific', 'attributes': [{ 'href': '/api/v2/root_gender/meta/id', 'fieldType': 'STRING', 'name': 'id', 'label': 'id', 'attributes': [], 'maxLength': 255, 'auto': false, 'nillable': false, 'readOnly': true, 'labelAttribute': false, 'unique': true, 'visible': true, 'lookupAttribute': false, 'isAggregatable': false }, { 'href': '/api/v2/root_gender/meta/label', 'fieldType': 'STRING', 'name': 'label', 'label': 'label', 'attributes': [], 'maxLength': 255, 'auto': false, 'nillable': false, 'readOnly': false, 'labelAttribute': true, 'unique': false, 'visible': true, 'lookupAttribute': true, 'isAggregatable': false }], 'labelAttribute': 'label', 'idAttribute': 'id', 'lookupAttributes': ['label'], 'isAbstract': false, 'writable': true, 'languageCode': 'en', 'permissions': ['UPDATE_DATA', 'DELETE_METADATA', 'AGGREGATE_DATA', 'ADD_DATA', 'READ_DATA', 'UPDATE_METADATA', 'READ_METADATA', 'COUNT_DATA', 'DELETE_DATA'] }, 'start': 0, 'num': 100, 'total': 2, 'items': [{ '_href': '/api/v2/root_gender/f', 'id': 'f', 'label': 'Female' }, { '_href': '/api/v2/root_gender/m', 'id': 'm', 'label': 'Male' }] })
```

Comment:
very long string makes test hard to maintain, only mock values needed, or if you really need a large data structure ( usually not the case of a unit test) move it to a human readable data structure ( js object)